controllers/sensor/controller.go

```diff
@@ -173,3 +175,25 @@ func (c *SensorController) runWorker() {
 	for c.processNextItem() {
 	}
 }
+
+func escalateError(name, namespace, reason, instanceID string, kubeClientset kubernetes.Interface) error {
```

Comment:
could we make this function public in a common package? I saw another function very similar to this

Reply:
this method was unnecessary and causing confusion, getting rid of it.